framework/web/migrations/m160313_153426_session_init.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+use yii\base\InvalidConfigException;
+use yii\web\DbSession;
+use yii\db\Migration;
+
+/**
+ * Initializes Session tables
+ *
+ * @author Misbahul D Munir <misbahuldmunir@gmail.com>
+ * @since 2.0.8
+ */
+class m160313_153426_session_init extends Migration
+{
+
+    /**
+     * @throws yii\base\InvalidConfigException
+     * @return DbSession
+     */
+    protected function getSession()
```

Comment:
Agree. But rbac migration use the same way. https://github.com/yiisoft/yii2/blob/master/framework/rbac/migrations/m140506_102106_rbac_init.php#L23-L30
For me. I prefer create table automatically when table does not exists like in Yii1. https://github.com/yiisoft/yii/blob/master/framework/web/CDbHttpSession.php#L158